src/test/appsCreate.spec.ts

```diff
@@ -0,0 +1,301 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import * as api from "../api";
+import { createAndroidApp, createIosApp, createWebApp } from "../appsCreate";
+import * as pollUtils from "../operation-poller";
+import { mockAuth } from "./helpers";
+
+const PROJECT_ID = "the-best-firebase-project";
+const OPERATION_RESOURCE_NAME_1 = "operations/cp.11111111111111111";
+const APP_ID = "appId";
+const IOS_APP_BUNDLE_ID = "bundleId";
+const IOS_APP_STORE_ID = "appStoreId";
+const IOS_APP_DISPLAY_NAME = "iOS app";
+const ANDROID_APP_PACKAGE_NAME = "com.google.packageName";
+const ANDROID_APP_DISPLAY_NAME = "Android app";
+const WEB_APP_DISPLAY_NAME = "Web app";
+
+describe("appsCreate", () => {
+  let sandbox: sinon.SinonSandbox;
+  let apiRequestStub: sinon.SinonStub;
+  let pollOperationStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+    mockAuth(sandbox);
+    apiRequestStub = sandbox.stub(api, "request");
+    pollOperationStub = sandbox.stub(pollUtils, "pollOperation");
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("createIosApp", () => {
+    it("should resolve with app data if it succeeds", async () => {
+      const expectedAppMetadata = {
+        appId: APP_ID,
+        displayName: IOS_APP_DISPLAY_NAME,
+        bundleId: IOS_APP_BUNDLE_ID,
+        appStoreId: IOS_APP_STORE_ID,
+      };
+      const createIosAppStub = createIosAppApiStub().resolves({
```

Comment:
Similar to the other PR, can these tests be adjusted so they're more readable?

Reply:
Done